src/main/java/org/apache/sysds/common/Builtins.java

```diff
@@ -173,6 +173,7 @@
 	TAN("tan", false),
 	TANH("tanh", false),
 	TRACE("trace", false),
+	TO_ONE_HOT("to_one_hot", true),
```

Comment:
This is a very Pythonic way of naming the function, I Just looked at the other function definitions and it is not very consistent in the naming. But the few fundamental functions taken from R is named using camelCase. I would suggest using that.
Any opinions @Shafaq-Siddiqi , is there an official decision?

Reply:
I have got no particular opinion on this, but I found a function called `one_hot` in an R package: https://www.rdocumentation.org/packages/mltools/versions/0.3.5/topics/one_hot 

Just let me know your final decision and I will change it accordingly.

Reply:
After discussing this with Matthias, we think that we should try to stick to camelCase. The same goes for the parameters. for the internal enum you can keep it as TO_ONE_HOT.

Reply:
Alright 👍 I just updated the naming in the most recent commit.